chemprop/v2/cli/utils/utils.py

```diff
@@ -75,3 +75,14 @@ def validate_loss_function(
                 f"Unknown readout function! got: {readout_ffn}. "
                 f"Expected one of: {tuple(readout.ReadoutRegistry.values())}"
             )
+
+def column_str_to_int(
+    columns: list,
+    header: list,
+    ) -> list:
+    if columns is None:
+        return None
+    if type(columns[0]) == str:
+        columns = [i for i,e in enumerate(header) if e in columns]
+    assert type(columns[0]) == int
```

Comment:
```suggestion
    if all(isinstance(col, str) for col in columns):
        columns = [i for i, name in enumerate(header) if name in columns]
    if not all(isinstance(col, int) for col in columns):
        raise ValueError #TODO: add message
```
I'm assuming that `columns` comes in as one of  `None | list[str], list[int]` and if it's `list[str]` then it gets converted to `list[int]`?